entgql/schema_test.go

```diff
@@ -80,6 +81,9 @@ type Todo {
   status: Status!
   priority: Int!
   text: String!
+  parent: Todo
+  childrenConnection: [Todo!] @goField(name: "children", forceResolver: false)
```

Comment:
Is it a wrong naming or a bug?

Reply:
Its just wrong naming in the schema then it effect the test, isnot a bug